doc/src/tutorials/biomechanics/biomechanics.rst

```diff
@@ -0,0 +1,595 @@
+acting on the two points.
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context:
+   :nofigs:
+
+   >>> pprint.pprint(lpathway.to_loads(-k*x - k*v))
+   [Force(point=O, force=(k*v(t) + k*x(t))*x(t)/sqrt(x(t)**2)*N.x),
+    Force(point=P, force=(-k*v(t) - k*x(t))*x(t)/sqrt(x(t)**2)*N.x)]
+
+Pathways can be constructed with any arbitrary geometry and any number of
+interconnected particles and rigid bodies. An example, a more complicated
+pathway is an :obj:`~sympy.physics.mechanics.pathway.ObstacleSetPathway`. You
+can specify any number of intermediate points between the two pathway endpoints
+which the actuation path of the forces will follow along. For example, if we
+introduce two points fixed in :math:`N` then the force will act along a set of
+linear segments connecting :math:`O` to :math:`Q` to :math:`R`: then to
+:math:`P`. Each of the four points will experience resultant forces. For
+simplicity we show the effect of only the spring force.
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context:
+   :nofigs:
+
+   >>> Q, R = me.Point('Q'), me.Point('R')
+   >>> Q.set_pos(O, 1*N.y)
+   >>> R.set_pos(O, 1*N.x + 1*N.y)
+   >>> opathway = me.ObstacleSetPathway(O, Q, R, P)
+   >>> opathway.length
+   sqrt((x(t) - 1)**2 + 1) + 2
+   >>> opathway.extension_velocity
+   (x(t) - 1)*Derivative(x(t), t)/sqrt((x(t) - 1)**2 + 1)
+   >>> pprint.pprint(opathway.to_loads(-k*opathway.length))
+   [Force(point=O, force=k*(sqrt((x(t) - 1)**2 + 1) + 2)*N.y),
+    Force(point=Q, force=- k*(sqrt((x(t) - 1)**2 + 1) + 2)*N.y),
+    Force(point=Q, force=k*(sqrt((x(t) - 1)**2 + 1) + 2)*N.x),
+    Force(point=R, force=- k*(sqrt((x(t) - 1)**2 + 1) + 2)*N.x),
+    Force(point=R, force=k*(sqrt((x(t) - 1)**2 + 1) + 2)*(x(t) - 1)/sqrt((x(t) - 1)**2 + 1)*N.x - k*(sqrt((x(t) - 1)**2 + 1) + 2)/sqrt((x(t) - 1)**2 + 1)*N.y),
+    Force(point=P, force=- k*(sqrt((x(t) - 1)**2 + 1) + 2)*(x(t) - 1)/sqrt((x(t) - 1)**2 + 1)*N.x + k*(sqrt((x(t) - 1)**2 + 1) + 2)/sqrt((x(t) - 1)**2 + 1)*N.y)]
+
+TODO : if you subs(x:1) then it is clear that the forces are acting in the
+right directions. But Force.subs() doesn't exist. to_loads could take a subs=
+kwarg.
+
+You can create your own pathways by subclassing
+:obj:`~sympy.physics.mechanics.pathway.PathwayBase`.
+
+Wrapping Geometries
+===================
+
+It is common for muscles to wrap over bones, tissue, or organs. We have
+introduced wrapping geometries and associated wrapping pathways to help manage
+their complexities. For example, if two pathway endpoints lie on the surface of
+a cylinder the forces act along lines that are tangent to the geodesic
+connecting the two points at the endpoints.
+
+:obj:`~sympy.physics.mechanics.wrapping_geometries.WrappingCylinder`
+:obj:`~sympy.physics.mechanics.pathway.WrappingPathway`
+
+TODO : "endpoints" seems like a better name than "attachements" for pathways
+given that pathway is in mechanics and not biomechanicis.
```

Comment:
We could do this, however, it would require refactoring all `PathwayBase` concrete classes. The `ObstacleSet` pathway would require endpoints, but it would also require intermediate points. The reason I chose to use `attachments` was so that for all pathway a sequence of `Point`s could be passed as the two "endpoints" would be chosen as the first and last of these. Moving to `endpoints` would mean that for `ObstacleSetPathway` we'd need to do something like:

```python
# Assume obstacle set pathway goes p1 -> p2 -> p3 -> p4
p1, p2, p3, p4 = Point('p1'), Point('p2'), Point('p3'), Point('p4')
pathway = ObstacleSetPathway(p1, p4, intermediate_points=[p3, p4])
```

Reply:
Ok, I'll probably keep the language in the text here with "endpoint" for these specific cases so I can distinguish among the intermediate points.